client/src/modules/patients/visits/modals/transfer.modal.js

```diff
@@ -0,0 +1,34 @@
+angular.module('bhima.controllers')
+  .controller('PatientTransferModalController', PatientTransferModalController);
+
+PatientTransferModalController.$inject = [
+  'VisitService', '$uibModalInstance', 'NotifyService', 'params',
+];
+
+function PatientTransferModalController(Visits, Modal, Notify, Params) {
+  const vm = this;
+
+  vm.details = {
+    display_name : Params.patient_display_name,
+    location : Params.location,
+  };
+
+  // expose action methods
+  vm.cancel = Modal.close;
+
+  vm.onBedRoomSelect = bed => {
+    vm.bed = bed;
+  };
+
+  vm.transfer = () => {
+    if (!vm.bed) { return; }
+
+    Visits.transfer(Params.patient_uuid, Params.patient_visit_uuid, vm.bed)
+      .then(transfered => {
+        if (!transfered) { return; }
+
+        Notify.success('PATIENT_RECORDS.TRANSFER.SUCCESSFULLY_TRANSFERED');
+        Modal.close(true);
+      });
```

Comment:
Should we do something if an error occurs?

Also, is there a loading indicator for this operation?  It looks like it might cause the user to wait for a bit.